x-pack/plugins/secops/public/components/timeline/body/__snapshots__/index.test.tsx.snap

```diff
@@ -481,1225 +481,2189 @@ Can be one or multiple IPv4 or IPv6 addresses.",
+            ],
+            "name": "zeek-sanfran",
+          },
+          "http": null,
+          "network": null,
+          "process": Object {
+            "args": Array [
+              "gpgconf",
+              "--list-dirs",
+              "agent-socket",
+            ],
+            "executable": "/usr/bin/gpgconf",
+            "name": "gpgconf",
+            "pid": 5402,
+            "ppid": 5401,
+            "title": "gpgconf --list-dirs agent-socket",
+            "working_directory": "/",
+          },
+          "source": null,
+          "suricata": null,
+          "timestamp": "2019-03-13T05:42:11.815Z",
+          "url": null,
+          "user": Object {
+            "name": "johnson",
+          },
+          "zeek": null,
         },
-        "zeek": null,
       },
       Object {
         "_id": "21",
-        "auditd": Object {
-          "data": Object {
-            "acct": null,
-            "op": "login",
-            "terminal": "/dev/pts/0",
-          },
-          "result": "success",
-          "session": "14",
-          "summary": Object {
-            "actor": Object {
-              "primary": "alice",
-              "secondary": "alice",
+        "data": Array [
+          Object {
+            "field": "@timestamp",
+            "value": Array [
+              "2019-03-14T22:30:25.527Z",
+            ],
+          },
+          Object {
+            "field": "event.category",
+            "value": Array [
+              "user-login",
+            ],
+          },
+          Object {
+            "field": "host.name",
+            "value": Array [
+              "zeek-london",
+            ],
+          },
+          Object {
+            "field": "source.ip",
+            "value": Array [
+              "8.42.77.171",
+            ],
+          },
+          Object {
+            "field": "user.name",
+            "value": Array [
+              "root",
+            ],
+          },
+        ],
+        "ecs": Object {
+          "_id": "21",
+          "auditd": Object {
+            "data": Object {
+              "acct": null,
+              "op": "login",
+              "terminal": "/dev/pts/0",
             },
-            "how": "/usr/sbin/sshd",
-            "message_type": null,
-            "object": Object {
-              "primary": "/dev/pts/0",
-              "secondary": "8.42.77.171",
-              "type": "user-session",
+            "result": "success",
+            "session": "14",
+            "summary": Object {
+              "actor": Object {
+                "primary": "alice",
+                "secondary": "alice",
+              },
+              "how": "/usr/sbin/sshd",
+              "message_type": null,
+              "object": Object {
+                "primary": "/dev/pts/0",
+                "secondary": "8.42.77.171",
+                "type": "user-session",
+              },
+              "sequence": null,
             },
-            "sequence": null,
           }
```

Comment:
I vote we delete this shallow test at this point. Github is not showing it due to its size and it provides little value at this level.